gms-go/generate.go

```diff
@@ -1,3 +1,3 @@
 package main
 
-//go:generate protoc -I./../proto -I. --go-grpc_out=paths=source_relative:./../proto --go_out=paths=source_relative:./../proto ./../proto/chat_service.proto
+//go:generate protoc -I./../proto -I. --go_out=paths=source_relative:./rpc ./../proto/chat_service.proto
```

Comment:
No need for separate output on `grpc service` generator

and we can directly output it to the `rpc` directory instead of moving it out manually from the Makefile